src/Columns/ColumnUnique.h

```diff
@@ -60,6 +60,8 @@ class ColumnUnique final : public COWHelper<IColumnUnique, ColumnUnique<ColumnTy
                                                                      size_t max_dictionary_size) override;
     size_t uniqueInsertData(const char * pos, size_t length) override;
     size_t uniqueDeserializeAndInsertFromArena(const char * pos, const char *& new_pos) override;
+    bool isEmpty() const override { return (column_holder->size() > numSpecialValues()); }
```

Comment:
It is suspicious. Should not it be `=` or `!(column_holder->size() > numSpecialValues())`?

Reply:
Oh yes! a stupid mistake, which would cause one more time trans index build.